docs/content.zh/docs/try-flink/table_api.md

```diff
@@ -28,51 +28,49 @@ under the License.
 
 # 基于 Table API 实现实时报表
 
-Apache Flink offers a Table API as a unified, relational API for batch and stream processing, i.e., queries are executed with the same semantics on unbounded, real-time streams or bounded, batch data sets and produce the same results.
-The Table API in Flink is commonly used to ease the definition of data analytics, data pipelining, and ETL applications.
+Apache Flink 提供 Table API 作为批流统一的关系型 API，例如，相同语法的查询在无界的实时流数据或者有界的批数据集上执行会得到一致的结果。  
+Table API 在 Flink 中常被用于简化数据分析、数据流水线以及 ETL 应用的定义。
```

Comment:
Flink 的 Table API 可以简化数据分析、构建数据流水线以及 ETL 应用。

Reply:
- Keep the 'Table API' at the front just because follow the context of previous line,like 'Flink offer Table API ... and Table API ...'.
- Keep 'definition' be translated maybe more clear?
>Flink 的 Table API 可以简化 __定义__ 数据分析、构建数据流水线以及 ETL 应用。

or 

>Flink 的 Table API 可以简化数据分析、构建数据流水线以及 ETL 应用的 __定义__。

Reply:
Flink 的 Table API 可以简化数据分析、构建数据流水线以及 ETL 应用的 定义。

Reply:
Also not sure is 'Flink 的 Table API 可以为数据分析、构建数据流水线以及 ETL 应用提供语法糖。'  a good free translation?
Otherwise we can keep the directly translate 'Flink 的 Table API 可以简化数据分析、构建数据流水线以及 ETL 应用的定义' ,because it is also make sence.

Reply:
> Also not sure is 'Flink 的 Table API 可以为数据分析、构建数据流水线以及 ETL 应用提供语法糖。' a good free translation? Otherwise we can keep the directly translate 'Flink 的 Table API 可以简化数据分析、构建数据流水线以及 ETL 应用的定义' ,because it is also make sence.

@ChengkaiYang2022 Hi,there.Do you think '语法糖' is OK?

Reply:
@Mrzyxing I think 'Flink 的 Table API 可以简化数据分析、构建数据流水线以及 ETL 应用的定义' is better.